README.md

```diff
@@ -0,0 +1,82 @@
+# tag-amd
+
+Transform an input into a tag container.
+
+## Dependencies
+
+tag-amd depends on an AMD loader (we recommend [async-define](http://elo7.github.io/async-define/)) and on [doc-amd](http://elo7.github.io/doc-amd/).
+
+## Installation
+
+You can use either `bower` or `npm` (preferred) to install it into your project:
+
+```
+bower install tag-amd
+npm install elo7-tag-amd
+```
+
+In the download package, there will be a JavaScript file (`tag.js`) and a CSS file (`tag.css`). The JavaScript file is not minified, but we strongly recommend minifying it before production usage. The CSS file is optional; you may use it as a reference for your own styling.
```

Comment:
It has a minified version on the dist folder, how about to change this comment and generate a minified version (js and css)?

@tisvasconcelos